python/Cargo.toml

```diff
@@ -13,6 +13,9 @@ keywords = ["deltalake", "delta", "datalake", "pandas", "arrow"]
 name = "deltalake"
 crate-type = ["cdylib"]
 
+[package.metadata.maturin]
+name = "deltalake._internal"
```

Comment:
Thanks! I was thinking we should rename this as well to indicate it's not part of the public API.